### PR TITLE
19 cloudinary implmentation

### LINKED ISSUE #19: cloudinary implmentation gettign the art

### PR DESCRIPTION
**Pull request by wwormseason**
whao images
also i changed up the game view to allow for submissions whiel retainign the preview function
thats it.

### CODE REVIEW

File: frontend/src/components/GamePage/GameArt.vue [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+const correct = ref(false)
+
+// const loading = ref(false)
+
+const getArt = async () => {
+  // loading.value = true
+  isVisible.value = false
+  artPieces.value = [await artStore.getRandomArt('human'), await artStore.getRandomArt('ai')]
+  if (artPieces.value.some((el) => el === null)) {
+    alert('Failed to fetch art, please ')
```
Comment (whydoesthecodenotwork) [inline frontend/src/components/GamePage/GameArt.vue]: please

**Review by whydoesthecodenotwork — APPROVED** [review]
sure??

**Comment by wwormseason** [comment]
> sure??

U SOUND UNSURE???? WHY

**Comment by whydoesthecodenotwork** [comment]
i do not want to be held liable for any damages that may result from the changes in this pr

**Pull request merged**